OpenRA.Mods.Common/Activities/Air/TakeOff.cs

```diff
@@ -30,12 +30,7 @@ public override Activity Tick(Actor self)
 			if (NextActivity == null)
 				self.CancelActivity();
 
-			var reservation = aircraft.Reservation;
-			if (reservation != null)
-			{
-				reservation.Dispose();
-				reservation = null;
-			}
+			aircraft.UnReserve();
```

Comment:
This will now null out the `Reservation` field on `aircraft`, rather than the `reservation` local. This is intended to be a fix.